### PR TITLE
Filter displayed items

### PR DESCRIPTION
**Pull request by up2code3**
Title and Date Search bar for found items 

note, date is year-month-day example 2024-19-04

### CODE REVIEW

**Review by royemosby — CHANGES_REQUESTED** [review]
The search functionality works great! I think it would be a good idea if you displayed something along the lines of "No search results" when there are 0 items to display. At a minimum you can create a 2em paragraph with centered text. That would be GTG but you can do something more elaborate if you like!

File: src/pages/Items.jsx [hunk 1]
```diff
@@ -99,27 +99,48 @@ const reportedItems =
...
+        <div className="searchContainer">
+          <input
+            className="searchField"
+            type="text"
+            placeholder="Search: title or date"
+            value={filter}
+            onChange={handleFilterChange}
+          />
+        </div>
+        <div className="reportedItemsContainer">
```
Comment (alejandroo-pc) [inline src/pages/Items.jsx:129]: This is a great approach! When we implement logic connecting the backend it should be an easy transition.

File: src/styles/Items.css [hunk 2]
```diff
@@ -34,14 +34,30 @@
 text-decoration: none;
 }
 
-
-
-
 .dateLostFound {
```
Comment (alejandroo-pc) [inline src/styles/Items.css:37]: good use of descriptive classnames, this will help scope styling down to specific components!

**Comment by up2code3** [comment]
should be good to go.

**Review by royemosby — APPROVED** [review]
👍

**Pull request merged**